app/helpers/SessionAnswersHelper.scala

```diff
@@ -262,4 +264,28 @@ class SessionAnswersHelper @Inject()() extends ImplicitDateFormatter {
       )
     )
   }
+
+  def getPreviousUploadsFileNames()(implicit request: UserRequest[_]): Future[String]= {
+    for {
+      previousUploads <- uploadJourneyRepository.getUploadsForJourney(request.session.get(SessionKeys.journeyId))
+    } yield {
+      val previousUploadsFileName = previousUploads.map(_.map(file => file.uploadDetails.map(details => details.fileName)))
+      previousUploadsFileName.getOrElse(Seq.empty).collect {
+        case Some(x) => x
+      }.mkString(", ")
+    }
+  }
+
+  def getContentWithExistingUploadFileNames(reasonableExcuse:String)(implicit request: UserRequest[_], messages: Messages): Future[Seq[(String, String, String)]]={
+    if(!reasonableExcuse.equals("other")){
```

Comment:
At's currently fail as I think its not checking files uploaded for obligation appeals you will want to add another check like
```suggestion
    if(!reasonableExcuse.equals("other") && !request.session.get(SessionKeys.isObligationAppeal).contains("true") {
```

Reply:
I haven't validated this change will make the AT's pass, but logically it should work

Reply:
> At's currently fail as I think its not checking files uploaded for obligation appeals you will want to add another check like

Updated the check to include Obligation Appeals Journey